app/src/main/java/com/framgia/soundcloud/screen/home/offline/OfflineViewModel.java

```diff
@@ -1,16 +1,37 @@
 package com.framgia.soundcloud.screen.home.offline;
 
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.content.Context;
+import android.database.Cursor;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.net.Uri;
+import android.provider.MediaStore;
```

Comment:
ERROR: Unused import: android.provider.MediaStore. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports